verification/src/transaction_verifier.rs

```diff
@@ -622,14 +588,20 @@ impl<'a, DL: HeaderProvider> SinceVerifier<'a, DL> {
         self.block_median_time(&parent_hash)
     }
 
+    fn parent_block_time(&self, block_hash: &Byte32) -> u64 {
```

Comment:
This fn name can be easily misinterpreted as getting the timestamp of the parent block, suggest to remove this fn, we can use the inline call `self.data_loader.get_header(block_hash).expect().timestamp()` directly